test/common/index.js

```diff
@@ -74,13 +77,21 @@ if (process.argv.length === 2 &&
     const args = process.execArgv.map((arg) => arg.replace(/_/g, '-'));
     for (const flag of flags) {
       if (!args.includes(flag) &&
-          // If the binary was built without-ssl then the crypto flags are
-          // invalid (bad option). The test itself should handle this case.
-          hasCrypto &&
           // If the binary is build without `intl` the inspect option is
           // invalid. The test itself should handle this case.
           (process.features.inspector || !flag.startsWith('--inspect'))) {
-        throw new Error(`Test has to be started with the flag: '${flag}'`);
+        console.log(
+          'NOTE: The test started as child_process using these flags:',
```

Comment:
This sounds weird to me. Maybe either

> NOTE: The test started as a child_process using these flags:

or if you want to keep it short

> NOTE: Test started as child_process using these flags:

? (Not a native English speaker, though 🤷‍♂️.)

Reply:
@Trott @vsemozhetbyt PTAL

Reply:
Not a native English speaker either, sorry. "as a child_process" seems OK, but I may be wrong.

Reply:
I think @tniessen's first suggestion is the most "natural" sounding:  "The test started as a child_process..."

Reply:
Thanks for the feedback!